superset/assets/javascripts/dashboard/components/SliceAdder.jsx

```diff
@@ -43,7 +45,7 @@ class SliceAdder extends React.Component {
   }
 
   onEnterModal() {
-    const uri = '/sliceaddview/api/read?_flt_0_created_by=' + this.props.dashboard.curUserId;
+    const uri = '/sliceaddview/api/read?_flt_0_created_by=' + this.props.userId;
```

Comment:
why not use string template literals? I think they're generally preferred over concatenation. should have a lint rule for this